README.md

```diff
@@ -65,7 +65,7 @@ Cedar represents principals, resources, and actions as entities. An entity has a
 
 Now, let's test our policy with the CLI
 ```rust
- cargo run  authorize \
+ cargo run cedar authorize \
```

Comment:
It should be `cargo run authorize` or `path_to_target/cedar authorize`

Reply:
fixed - the double spaces looks like something was mising